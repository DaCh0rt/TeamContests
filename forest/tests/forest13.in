945 851
-5223 -8759
4465 131
5380 2073
-7838 6809
4485 -979
-1525 6299
47 7042
-6395 9020
9514 5363
-2433 4767
-3044 -8495
9038 -9328
-8765 -1450
3394 -1670
6088 2539
-7518 -4966
-8401 -9931
-9734 -8095
-9265 2008
-7808 7210
9961 -8668
6049 -5142
-9644 -9061
6980 -7809
-5532 8973
9363 -5020
-3121 9424
7029 9616
-7364 -3739
-5353 8839
-9742 3453
-4628 2994
-1164 -5294
2195 -3576
-5497 -7378
9897 -4581
-622 -8346
1938 1824
-7564 3407
7297 -2153
3723 6053
-8325 -9804
-3193 2397
-7103 2420
3914 -6338
-5997 206
4648 1308
2252 8404
-8041 -7591
7314 -8443
-9872 -7781
-9412 1373
-8964 -4174
5642 -3591
4695 5748
-3971 -8597
-8280 4134
1855 -5010
3794 -3600
-607 5079
9655 4076
-9072 -3590
-3172 -2656
-5997 -8081
-7913 7045
-9807 7164
-9751 -1721
6845 1990
4202 3303
6349 688
3009 -6801
-2298 5294
-450 -4950
-8786 -2597
-4441 -2491
4720 4117
6110 4301
-8739 1512
-8104 7105
-2564 3645
-9143 -3826
9792 9282
9358 7673
2942 -5695
4480 5612
-4286 1132
-4639 2948
-1837 -7817
-3204 8625
-9460 4975
-4054 5225
2390 -8635
-6437 6659
1547 6674
-2882 -274
7394 9513
-8487 -9439
7661 -6559
-7859 -8985
-9662 7311
-8454 155
-9650 1272
-6856 -4526
-6404 9980
6069 9970
7732 4045
-2209 4117
5586 963
5098 6577
-7365 -2215
4750 -3814
-6105 8902
-255 -7327
8473 8301
-2102 6853
5656 3144
8719 -1495
3319 -1799
-7695 5894
-5619 7142
-816 -7011
5975 7354
-2509 -9532
4471 -1563
-9137 -8244
-7612 -7278
-6016 5522
-5927 -9560
4959 -1640
2445 5087
8509 577
3253 -2138
1434 -2109
2360 7304
-9331 -7270
7893 -2207
5858 9581
-9075 1419
-8895 -4099
-6933 5401
9706 7399
8657 9420
377 8675
5419 7357
8641 -2393
-6685 415
-682 -5132
-9682 -294
8957 -2945
-4462 -4096
-9836 -5090
-300 2805
-9734 6003
5591 -3001
2284 -7202
-3237 -2779
7053 -4664
9520 5621
1569 -1544
-9717 -6399
3221 7011
8211 1787
7400 -2875
-3140 -9483
-985 8555
-5883 -8259
-739 1321
-8664 1161
3432 -6290
6884 -4544
-9976 -1216
9976 -9633
-701 2338
-8857 3041
2276 274
6001 294
-975 1681
-1715 8632
-2982 782
9122 -4573
-5198 196
2575 -4337
5479 6751
-7305 -7123
-1370 -6609
8894 -703
138 -7508
9969 6373
1996 -6781
-9141 2714
-1498 5736
-1618 -8552
-2653 -9884
-4476 -583
4385 -3992
-3799 -2909
6369 9351
4381 4845
-7915 4502
-9788 -8865
-6522 7238
8320 881
-9058 1113
7862 1265
1803 9715
-3015 -8852
8912 -8713
5764 4281
-3437 4017
1088 -3432
795 -2649
-1161 -2050
103 2699
1415 7775
8401 8888
-1441 -9122
-1523 -3048
9215 -3879
4059 -4590
3885 -2078
-416 3041
7345 -9981
-7568 -8040
-4144 6191
2788 -4528
7775 -3653
-8527 -3219
9626 -4140
3489 1495
-5288 -6682
86 6954
6406 1873
9802 9371
-405 6892
3767 1651
-8451 -9307
918 1580
-6161 -8541
-7385 4850
8800 -8992
-7160 3295
-8449 -3895
-329 -6595
3884 -3010
7015 -5110
-8495 -7876
-7580 5352
6123 -7137
-771 -21
-999 -9969
7671 1496
-4980 1890
6482 2003
-1191 -4113
1258 8629
-9356 -3996
-8569 2868
4109 -4338
9792 -8509
8914 2624
-6900 6073
2907 -3350
6982 -450
5458 1481
-2768 8727
7376 -1377
-653 8966
-7891 5030
-2066 -5313
8402 6041
-7 3878
-3122 -798
-3551 -8079
-9845 4296
8475 4185
-4205 -9757
154 398
-6536 6888
8714 4741
-6681 -8666
-7355 -3222
-9392 -274
-1761 1289
-6119 -4458
6972 6335
6335 6343
805 3034
8485 5672
9385 6476
-323 -4442
-6440 4008
6490 -1157
9945 5554
-6424 -572
-3506 9827
-393 -8615
8399 -1359
6778 -2756
736 6241
-6847 -8044
-2663 635
4074 -3550
-5514 -9890
858 5334
6812 -997
-1917 1437
-7566 2002
7451 3765
9971 -5448
8800 -613
7640 -3202
-8742 8324
4557 -2746
-1995 9429
3852 8085
5115 -3918
-7650 -8983
-4420 -8854
5562 -2773
3144 5842
-578 -6462
8522 -5749
3865 -1686
-7372 7044
8388 -1079
4228 2684
36 -9854
1656 -5202
-8885 -1414
-5052 7199
-5969 6758
-3203 3367
-7796 8039
-8882 9901
-777 -2370
-7514 3899
-7380 -5961
-3580 8463
5556 7380
5186 -1526
-797 -1259
-6460 -2964
-9325 8186
-6428 -2026
-9453 -5443
6959 -6283
5332 7171
7206 -6228
9255 5319
6336 6810
6577 2693
9881 -4660
-2748 5686
3092 -6280
7243 -7212
4686 9916
8 -5488
-7256 4415
-4705 7722
-8722 2743
7952 7434
8717 3421
-1984 7127
-8665 -1695
5795 -5103
1096 6901
9589 -6215
-4949 -185
9380 -3806
-5640 7868
-5690 6578
-1799 -7901
735 -421
3987 -5770
-7437 -5745
-3757 3871
-2972 -1208
-5572 712
-3854 -7345
-400 7019
194 -7887
-5617 3315
8416 371
7736 3808
1454 1340
-8951 90
-5269 -1656
-7064 -5955
6874 -7538
-2803 -5497
6064 -2595
1233 -6147
4110 -3728
3508 -5559
5776 2626
-9758 2012
1193 292
1822 -6846
-6658 1215
3532 4926
-9496 7766
-263 -6478
-7334 7994
9647 1541
8388 7002
3472 4140
608 -8386
9004 -6749
-1549 -2506
-3301 6308
8843 -1733
-9538 1392
7411 8532
9437 1833
8984 -3440
-9649 -2774
-3909 -2068
-8532 -2360
3943 1298
-9083 -835
9026 7339
-4163 -7132
-1603 -900
2107 -9319
-8629 -7780
-5809 3823
6118 9958
7413 -5060
-159 1375
-6634 -8981
439 -8156
5891 2090
581 -5490
-6302 -5270
-9370 -5216
8322 2096
-5931 -2103
-9710 432
-4756 109
9916 7800
360 5231
-4154 6217
7338 9540
-6063 6915
2466 -200
5014 -8220
-6897 -6844
-830 -6382
385 7805
2110 -2029
-1139 -4115
-874 6872
-6732 3864
-4857 -6654
-3644 6820
-8223 -6768
-6051 800
-5842 -7870
5998 689
-5396 7194
8441 -5243
-9128 -291
9795 8493
2109 9885
7347 -5737
-680 -929
4219 7989
-7174 -9746
-7946 2310
289 4875
-3547 3948
-7539 -4585
-6143 3424
-6194 -8335
-9707 -6624
-1343 8376
-7027 2842
-7653 -1506
-386 -2327
-2548 3853
-6676 922
-8429 -1098
-8776 -4289
129 8362
4721 3595
4744 1496
-1029 -5757
580 -115
7735 -8137
413 2261
-660 -2258
6119 149
-4137 1403
7206 -6715
4061 -2398
-3237 -6046
-5165 8670
2088 -9725
9106 3445
-4445 7232
-3684 5881
9555 -5709
-9407 4087
-1241 -853
7359 -1834
9019 -6789
3584 5053
-9112 -2207
9646 1874
4209 4828
-5165 -7684
1984 -3556
9277 371
-7860 -2060
920 9537
6182 -2197
-6446 4373
-2712 -752
-3652 -5385
-6856 -9505
8653 -9142
6438 4318
-1541 9805
-4182 1956
9741 -3392
-946 -7205
1659 -9934
6737 -8721
-4789 -3714
-9033 -7113
-3714 -4797
-9363 -2952
1290 -1071
5560 -6971
-9495 1724
828 -8598
1065 7381
2770 -8161
1085 4400
-3269 7290
-5225 -2413
-4956 -1322
-67 8734
8302 3996
3669 6358
2298 -897
-3405 2264
2558 2688
-4062 7588
9718 578
7702 2628
-6918 -8842
-7553 -2145
-2492 -4917
-2291 -1502
-5338 -5476
-1195 7340
2260 36
-1035 -3696
-2569 -5238
169 -4226
5292 -1274
1926 6867
5469 8019
-2335 7514
-1849 -2885
3664 2751
5682 -4745
-9660 -3287
-4846 -7636
8455 -3243
-6549 -1432
-8959 9037
-6674 9544
6990 -7237
-9739 5393
4697 3226
-4361 8156
-2013 2650
-2619 9469
5557 6195
-5726 -5289
-2930 -9940
-1822 -3570
8544 -678
-6506 7521
7519 1563
9605 3596
3054 8022
9665 -3537
9407 -4912
224 2830
5817 2040
6412 -2925
4320 5273
407 388
-2265 6009
8084 -1618
-1427 -835
-9578 -1108
4863 -5635
9276 -2161
9273 -5268
6174 9418
1107 4166
1140 -6700
7501 -478
8257 2599
-5539 9826
4871 -6598
-9282 3449
-182 -9158
4959 -4682
-4217 -4089
-7126 -2847
-8818 9291
2065 -7864
-4653 -5156
2214 4366
3935 4012
-8192 -6451
-7155 1701
-3210 3721
-9261 -7595
3152 4151
-8476 9382
-3313 3725
-807 3381
-9119 -4930
5183 -4899
1287 1794
-3441 3518
2081 -3702
-8068 44
-7103 -2867
5255 6247
9105 4910
6146 7983
-2873 -6200
-2135 -5390
-1089 -6685
-4772 2025
-5117 -1189
8918 -1341
8219 9691
4745 9735
-2944 -6582
8472 4914
4470 -883
4662 -9417
5258 1095
747 -138
8238 -6131
392 -1400
3871 343
-2458 3278
-4930 3972
-5669 206
-9342 -6093
6554 4724
-6092 6315
-342 -617
-596 4331
6040 4829
3100 4316
8528 218
-2006 4819
-7888 6671
-7989 9349
-6261 -5674
639 -8055
838 -968
4563 9324
6293 94
-3255 4336
9521 7198
-3031 4975
-7564 -5738
6162 -4691
-8841 7657
-8136 2719
8267 1893
3582 -9871
1021 7894
9678 8995
-23 9755
-9847 8359
-1750 -4797
5546 8728
-9360 3844
-152 -8859
812 -1752
-1821 3097
-7526 -2523
-6799 -3794
-7390 -11
7112 2595
591 -6531
7801 -6881
-540 -4572
-1493 -3725
-4088 4310
9558 -386
-8565 4960
-2766 2730
-8391 451
-7839 -681
4386 -4599
-6599 9970
-1083 7838
-8933 6082
1313 -5393
-6673 -8770
-5234 9155
8901 7289
-933 1055
-9844 -2470
6271 -883
-3015 5364
-105 -9344
5705 3811
7533 -9376
1335 -2004
2703 -3790
-3771 4357
7307 -190
276 9674
-1821 1514
-7991 2358
-8882 2744
2121 -1691
7843 7040
125 -4797
-9491 -5644
-3538 3802
-7767 4815
2422 -6808
3829 1070
-3086 -5456
-4117 -195
-6582 5905
-946 5605
-9303 4533
4416 2719
2765 6946
-6624 -8888
5277 -7146
5286 6276
-1075 1100
10 8874
-6737 1204
4667 -3772
9287 -708
-8127 -8765
4476 -7805
9907 9738
-117 -3386
7499 5181
-6076 7540
-2179 9161
-9525 -9524
630 613
4823 -6976
2706 3463
4063 -2227
1177 -6595
-2523 -4940
-7627 -8157
3328 -4106
8210 -3422
6745 5893
8578 8823
1212 3615
3475 -6339
2637 9727
-4204 -2206
-8926 -4458
1761 -2838
-7699 -4094
5410 8963
-4428 -6611
-1580 -2260
-9202 7300
-9305 5953
-680 8451
-5621 -3383
-753 1358
-7497 4829
6385 -7829
8231 9900
-6140 9765
9752 6908
2641 -7116
-5810 -8073
6614 4841
-2174 534
2857 -9807
1467 -7507
-3289 -7337
6654 4540
7571 -8319
8861 1392
7324 -8455
3367 -4623
3436 -3529
-8659 7676
-1431 -2796
5050 1564
-609 4784
110 2248
7627 3547
-5961 -3709
9622 -6330
-1385 -5041
9294 -8169
9568 -4806
-2214 -9423
4953 6431
-8001 -7531
5267 -2786
-737 852
9665 1044
4322 8538
-4384 4734
3528 -8120
3886 -7576
3012 3834
-4228 -4398
-7895 -7461
-7448 -6610
-7038 -1953
-4250 -8060
3821 4676
-3303 -5556
-1387 2076
-7981 9251
-7289 1988
2725 -4459
-9532 -1447
-5632 -7152
3903 -9341
562 1417
-95 -1929
4181 3960
2105 -5017
7012 6862
313 8081
3316 3792
7427 -9299
7443 -1588
-3565 2605
-8178 -8245
-9557 4095
-8347 2148
-4242 -9424
-6905 9649
723 -1591
-600 1344
-2753 -9730
-5001 5410
-761 4318
167 8329
3068 -5132
-9791 -8531
-7130 8622
-7793 9681
6966 -1161
-218 4178
-7391 1329
9449 -9262
2111 1234
-3266 9095
-6218 814
5991 3253
533 -6784
297 -7830
2520 -4662
5585 6632
8623 -8178
-1813 451
6134 4314
-443 1351
1730 -996
3306 -3721
-2852 -7549
3908 476
-6489 -4110
-6052 -2680
-6428 842
-4681 8886
-8173 -8575
-6707 -2811
-452 -5906
388 9953
-3393 -2421
3337 -3019
-758 7485
4216 -8432
4211 59
-1757 7571
1959 -8836
5337 7710
6930 -3329
-8861 114
9730 995
-624 5433
6168 4066
-6921 6279
-180 -856
-3783 4110
6825 -1204
5614 215
-6481 2770
-6659 -6367
-5535 2417
-96 -8282
-1665 -5124
-454 7635
-8673 -9795
4080 -6886
-5019 -7097
-3029 -4012
8307 -6170
4045 -692
3170 7932
-9470 -362
2411 -8122
-8368 2905
-211 4702
-3826 -8885
9150 6702
-4728 6823
-6745 8523
2620 7764
-4066 -9040
1304 7342
-3527 5786
-3095 -6627
-1543 4830
-2969 4752
4540 -9549
9549 1082
-1117 6676
-3662 -544
-8353 -2116
