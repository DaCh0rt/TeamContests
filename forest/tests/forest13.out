82870.76 462196278.75
